addOns/pscanrulesBeta/src/main/java/org/zaproxy/zap/extension/pscanrulesBeta/InformationDisclosureInURL.java

```diff
@@ -42,10 +43,19 @@
     private static final int PLUGIN_ID = 10024;
 
     private PassiveScanThread parent = null;
-    private static final String URLSensitiveInformationFile =
-            "xml/URL-information-disclosure-messages.txt";
+    public static final String URLSENSITIVEINFORMATIONDIR = "xml";
+    public static final String URLSENSITIVEINFORMATIONFILE =
+            "URL-information-disclosure-messages.txt";
     private static final Logger logger = Logger.getLogger(InformationDisclosureInURL.class);
-    private List<String> messages = null;
+    private static List<String> messages = null;
+    static Pattern emailAddressPattern =
+            Pattern.compile("\\b[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,4}\\b");
+    // CC Pattern Source:
```

Comment:
Was it?

Reply:
Are you asking if that's where the credit card pattern came from? If so, then yes, that's where we found it.

Reply:
It most likely was but after 7 years who knows.

Reply:
I just checked to make sure by going to the URL, it is still there :)